test/jdk/java/foreign/NativeTestHelper.java

```diff
@@ -105,4 +106,8 @@ public static MemoryAddress allocateMemory(long size) {
             throw new IllegalStateException(ex);
         }
     }
+
+    public static NativeSymbol findNativeOrThrow(Class<?> clazz, String name) {
```

Comment:
note: if this was an _instance_ method, then I think you could get rid of the `clazz` parameter, and just use `this` ?

Reply:
good point, changed it